### DISCUSSION TITLE
Regenerating snapshots in connector fields

### DISCUSSION THREAD

**Discussion by siurna** [AntlerVC/firetable#737] [Support / Q&A] [answered]
Hello,
 is there a way to regenerate connector field-based value, when the value connected was changed?

 For example, I am building a backend for a small blog/news website application right now, and every time I change category/taxonomy information, I need manually unselect and select the value in all fields associated (like, category field in posts table, given the column is connected to category table).

 Is writing a cloud function the best way to work around this or can it be done somewhat automatically? Given that I dont want to alter any fields, just trigger a refresh.

Thanks!

**Comment by BeeBombshell** [comment]
Hi @siurna, I'm not sure I follow. The connector field is a dropdown field that lets you pick values from another table, API, or a static list. Are you trying to change the field value in the source table and are expecting these changes to be reflected in the previously selected connector field in another table?

Could you possibly provide a little more clarification? Something like a screen recording to see if we can help you better? 🙂

**Comment by harinij** [comment, marked as answer]
@siurna in case you are still looking for a solution for this, since the data stored once you pick an option is a snapshot of the other collection at that given point. If the original collection's data is changed then it's not reflected. One option is for you to have a derivative column/extension on the original table that can listen to for changes and updated the connected field.

We will look into adding this natively to the connector field in the future.
